src/Arcus.Template.WebApi/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2.7-alpine3.9 AS base
+WORKDIR /app
+EXPOSE 80
+
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2-stretch AS build
+WORKDIR /src
+COPY ["Arcus.Template.WebApi.csproj", ""]
+RUN dotnet restore "./Arcus.Template.WebApi.csproj"
+COPY . .
+WORKDIR "/src/."
+RUN dotnet build "Arcus.Template.WebApi.csproj" -c Release -o /app
```

Comment:
We can skip this and just use `dotnet publish` who implicitly does this

Reply:
I'm not sure that's entirely correct since the build fails with missing assembly files.

Reply:
Based on https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-publish I don't see the need to restore (explicitly mentioned) and given it has a `--no-build` option it's clear that it builds it anyway. What is your concern?

Reply:
My concert is that it doesn't build :).
I thought that publish was not exactly the same as building since publish will only build if any changes to the source was made and build will always re-compile the source.

Reply:
Try it otherwise locally on any project; I get the same result.

Reply:
I'm not sure what your concern is, the docs clearly indicate that you can flag it to not build and that the output is ready to run:

> The dotnet publish command's output is ready for deployment to a hosting system (for example, a server, PC, Mac, laptop) for execution.

It doesn't mention that a build up front is required?

> Try it otherwise locally on any project; I get the same result.

What do you mean with same result?

Reply:
If I try to publish a non-build-yet project locally it also fails.
I know that the docs says otherwise, but apparently I can't get the expected result (even on a fresh project that's not build for the first time).

Some question/discussions online also talks about this. There is a diff between the two apparently.

Reply:
Ok, fine for me than - Sorry about that!

Can we then just remove the restore if it's ok - Otherwise just proceed and resolve the comments

Reply:
Ok, I'll remove the restore.

Reply:
And just checked; the current Dockerfile (without restore) succeeds.

Reply:
Awesome, thanks!